### DISCUSSION TITLE
Add standard icon pack to get users started?

### DISCUSSION THREAD

**Discussion by RemcoSchrijver** [streamdeck-linux-gui/streamdeck-linux-gui#65] [Ideas / Feature Requests]
It might be nice to get a standard icon pack for users to get started creating setups. For example [Dashy](https://dashy.to/docs/icons/) allows for using many icon providers. And [Aegis](https://github.com/aegis-icons/aegis-icons) allows to download an icon pack in the UI. 

Something similar to this might be a nice inclusion.

**Comment by Data-007** [comment]
On our discord we have we have a  chat,  several posts there on icon packs
and where to get them. In the #development channel. https://discord.gg/zKhK8EYe

**Reply by RemcoSchrijver** [reply]
Ahh I see but what I am advocating for is an in app process of more easily loading icon packs/shipping certain icons packs like FA or Material.
